gym/spaces/dict.py

```diff
@@ -49,6 +55,23 @@ def __init__(
         seed: Optional[dict | int | seeding.RandomNumberGenerator] = None,
         **spaces_kwargs: Space,
     ):
+        """Constructor of `Dict` space.
+
+        This space can be instantiated in one of two ways: Either you pass a dictionary
+        of spaces to `__init__` via the ``spaces`` argument, or you pass the spaces as separate
+        keyword arguments (where you will need to avoid the keys ``spaces`` and ``seed``)
+
+        Example::
```

Comment:
I believe a line is required between example and the code block for the docstring to format correctly